src/convnets/inception.jl

```diff
@@ -176,21 +178,303 @@ See also [`inception3`](#).
+classifier(m::Inceptionv4) = m.layers[2]
+
+## Inception-ResNetv2
+
+function mixed_5b()
+    branch1 = Chain(conv_bn((1, 1), 192, 96)...)
+    branch2 = Chain(conv_bn((1, 1), 192, 48)...,
+                    conv_bn((5, 5), 48, 64; pad = 2)...)
+    branch3 = Chain(conv_bn((1, 1), 192, 64)...,
+                    conv_bn((3, 3), 64, 96; pad = 1)...,
+                    conv_bn((3, 3), 96, 96; pad = 1)...)
+    branch4 = Chain(MeanPool((3, 3); pad = 1, stride = 1),
+                    conv_bn((1, 1), 192, 64)...)
+    return Parallel(cat_channels, branch1, branch2, branch3, branch4)
+end
+
+function block35(scale = 1.0f0)
+    branch1 = Chain(conv_bn((1, 1), 320, 32)...)
+    branch2 = Chain(conv_bn((1, 1), 320, 32)...,
+                    conv_bn((3, 3), 32, 32; pad = 1)...)
+    branch3 = Chain(conv_bn((1, 1), 320, 32)...,
+                    conv_bn((3, 3), 32, 48; pad = 1)...,
+                    conv_bn((3, 3), 48, 64; pad = 1)...)
+    branch4 = Chain(conv_bn((1, 1), 128, 320)...)
+    return SkipConnection(Chain(Parallel(cat_channels, branch1, branch2, branch3),
+                                branch4, inputscale(scale; activation = relu)), +)
+end
+
+function mixed_6a()
+    branch1 = Chain(conv_bn((3, 3), 320, 384; stride = 2)...)
+    branch2 = Chain(conv_bn((1, 1), 320, 256)...,
+                    conv_bn((3, 3), 256, 256; pad = 1)...,
+                    conv_bn((3, 3), 256, 384; stride = 2)...)
+    branch3 = MaxPool((3, 3); stride = 2)
+    return Parallel(cat_channels, branch1, branch2, branch3)
+end
+
+function block17(scale = 1.0f0)
+    branch1 = Chain(conv_bn((1, 1), 1088, 192)...)
+    branch2 = Chain(conv_bn((1, 1), 1088, 128)...,
+                    conv_bn((1, 7), 128, 160; pad = (0, 3))...,
+                    conv_bn((7, 1), 160, 192; pad = (3, 0))...)
+    branch3 = Chain(conv_bn((1, 1), 384, 1088)...)
+    return SkipConnection(Chain(Parallel(cat_channels, branch1, branch2),
+                                branch3, inputscale(scale; activation = relu)), +)
+end
+
+function mixed_7a()
+    branch1 = Chain(conv_bn((1, 1), 1088, 256)...,
+                    conv_bn((3, 3), 256, 384; stride = 2)...)
+    branch2 = Chain(conv_bn((1, 1), 1088, 256)...,
+                    conv_bn((3, 3), 256, 288; stride = 2)...)
+    branch3 = Chain(conv_bn((1, 1), 1088, 256)...,
+                    conv_bn((3, 3), 256, 288; pad = 1)...,
+                    conv_bn((3, 3), 288, 320; stride = 2)...)
+    branch4 = MaxPool((3, 3); stride = 2)
+    return Parallel(cat_channels, branch1, branch2, branch3, branch4)
+end
+
+function block8(scale = 1.0f0; no_relu = false)
+    branch1 = Chain(conv_bn((1, 1), 2080, 192)...)
+    branch2 = Chain(conv_bn((1, 1), 2080, 192)...,
+                    conv_bn((1, 3), 192, 224; pad = (0, 1))...,
+                    conv_bn((3, 1), 224, 256; pad = (1, 0))...)
+    branch3 = Chain(conv_bn((1, 1), 448, 2080)...)
+    activation = no_relu ? identity : relu
```

Comment:
Why not directly pass in the activation as a function instead of a keyword?

Reply:
Yep, was a quirk in the way I wrote it initially before refactoring 🤦🏽 I'll change it up